src/QirRuntime/lib/QIR/bridge-rt.ll

```diff
@@ -108,6 +108,10 @@ declare i8* @quantum__rt_string_get_data(%"struct.QirString"* %str)
 declare i32 @quantum__rt_string_get_length(%"struct.QirString"* %str)
 
 
+;------------------------------------------------------------------------------
+; message
+;
+declare void @quantum__rt__message(%"struct.QirString"* %str)

```

Comment:
A minor suggestion (feel free to ignore): I would put this up with @quantum__rt__fail under classical.